src/test/resources/fixtures/createLegacyBeaconRequest.json

```diff
@@ -0,0 +1,150 @@
+      },
+      "uses": [
+        {
+          "pkBeaconUsesId": 6057,
+          "fkBeaconId": 6062,
+          "vesselName": "KAYCEE",
+          "homePort": "None",
+          "maxPersons": 10,
+          "officialNumber": "None",
+          "rssSsrNumber": "None",
+          "callSign": "VQAS7",
+          "imoNumber": "None",
+          "mmsiNumber": 235007399,
+          "fishingVesselPln": "None",
+          "hullIdNumber": "None",
+          "cg66RefNumber": "None",
+          "aodSerialNumber": "None",
+          "principalAirport": "None",
+          "bit24AddressHex": "None",
+          "aircraftRegistrationMark": "None",
+          "areaOfUse": "None",
+          "tripInfo": "None",
+          "rigName": "None",
+          "beaconPosition": "None",
+          "position": "None",
+          "localManagementId": "None",
+          "beaconNsn": "None",
+          "beaconPartNumber": "None",
+          "notes": "None",
+          "pennantNumber": "None",
+          "aircraftDescription": "None",
+          "survivalCraftType": "None",
+          "communications": "VHF/DSC",
+          "isMain": "Y",
+          "createUserId": 2889,
+          "createDt": "2015-03-24T11:35:11",
+          "updateUserId": 2889,
+          "updateDt": "2015-03-24T11:35:11",
+          "versioning": 0,
+          "useType": "Maritime",
+          "vesselType": "Pleasure",
+          "aircraftType": "None",
+          "landUse": "None",
+          "note": "Some notes"
+        },
+        {
+          "pkBeaconUsesId": 6058,
+          "fkBeaconId": 6062,
+          "vesselName": "KAKEE",
+          "homePort": "None",
+          "maxPersons": 10,
+          "officialNumber": "None",
+          "rssSsrNumber": "None",
+          "callSign": "VQAS7",
+          "imoNumber": "None",
+          "mmsiNumber": 235007399,
+          "fishingVesselPln": "None",
+          "hullIdNumber": "None",
+          "cg66RefNumber": "None",
+          "aodSerialNumber": "None",
+          "principalAirport": "None",
+          "bit24AddressHex": "None",
+          "aircraftRegistrationMark": "None",
+          "areaOfUse": "None",
+          "tripInfo": "None",
+          "rigName": "None",
+          "beaconPosition": "None",
+          "position": "None",
+          "localManagementId": "None",
+          "beaconNsn": "None",
+          "beaconPartNumber": "None",
+          "notes": "None",
+          "pennantNumber": "None",
+          "aircraftDescription": "None",
+          "survivalCraftType": "None",
+          "communications": "VHF/DSC",
+          "isMain": "N",
+          "createUserId": 2889,
+          "createDt": "2015-03-24T11:35:11",
+          "updateUserId": 2889,
+          "updateDt": "2015-03-24T11:35:11",
+          "versioning": 0,
+          "useType": "Maritime",
+          "vesselType": "Pleasure",
+          "aircraftType": "None",
+          "landUse": "None",
+          "note": "Some more notes"
+        }
+      ],
+      "owner": {
```

Comment:
On the etl side, I've changed this to `owners` and made it an array! To match what we saw and discussed around multiple owners